manager/integration/tests/common.py

```diff
@@ -3493,19 +3493,34 @@ def wait_deployment_replica_ready(apps_api, deployment_name,
     assert replicas_ready
 
 
-def create_and_wait_deployment(apps_api, deployment_manifest):
+def create_and_wait_deployment(apps_api, deployment_manifest, wait=True):
```

Comment:
Since this function name is "create_**and_wait**_deployment", it's weird to add this argument. You can create another function to create deployment only.
```
def create_and_wait_deployment(apps_api, deployment_manifest):
    create_deployment()
    wait_deployment_replica_ready()
    ......


def create_deployment(apps_api, deployment_manifest):
```

Reply:
You are right, this does look weird. Addressed.